android-sdk/android/src/main/kotlin/io/logto/sdk/android/auth/logto/LogtoAuthManager.kt

```diff
@@ -0,0 +1,36 @@
+package io.logto.sdk.android.auth.logto
+
+import android.annotation.SuppressLint
+import android.net.Uri
+
+internal object LogtoAuthManager {
+    private var logtoAuthCallbackUriScheme: String? = null
```

Comment:
if we just use fixed scheme then there's no need to define a var here

Reply:
updated. no need for this, for we capture this in the web-view